chromium_src/chrome/browser/upgrade_detector/upgrade_detector_impl.cc

```diff
@@ -14,11 +15,10 @@
 // especially because it is likely that Brave is just downloading an update in
 // the background. To work around this, we disable outdated build detection
 // until we also have background updates on macOS.
-#if BUILDFLAG(IS_MAC)
-#include "brave/browser/mac_features.h"
+#include "brave/browser/updater/features.h"
 
 #define BRAVE_UPGRADE_DETECTOR_IMPL_START_OUTDATED_BUILD_DETECTOR \
-  if (!brave::ShouldUseOmaha4()) {                                \
+  if (!brave::updater::ShouldUseOmaha4()) {                       \
```

Comment:
I think this should be controlled by whether background updates are enabled or not because even with Omaha 4 they could be disabled. I think there is also some other process that eventually sets this because I have seen it on older builds on mac.

Reply:
to be clear this is not meant to be a change for this PR